com.unity.uiwidgets/Runtime/painting/image_provider.cs

```diff
@@ -790,15 +790,32 @@ public class NetworkImage : ImageProvider<NetworkImage>, IEquatable<NetworkImage
     }
 
     public class FileImage : ImageProvider<FileImage>, IEquatable<FileImage> {
-        public FileImage(string file, float scale = 1.0f) {
+        public FileImage(string file, float scale = 1.0f, bool isAbsolutePath = false) {
```

Comment:
This change has been merged in previous pr ?

Reply:
yes, this change has been merged to `master` branch, but not the target of this PR, i.e., the `zxw/3d` branch